packages/office-ui-fabric-react/src/components/ContextualMenu/examples/ContextualMenu.Header.Example.tsx

```diff
@@ -83,7 +83,8 @@ export class ContextualMenuHeaderExample extends React.Component<{}, {}> {
             {
               key: 'Bing',
               text: 'Go to Bing',
-              href: 'http://www.bing.com'
+              href: 'http://www.bing.com',
```

Comment:
Even though it *probably* doesn't make a material difference for our examples, this and all URLs should probably be `https`. No need to change for this PR—we can do it separately—just wanted to point out while it's in front of me :)